src/Services/Gateways/BaseGateway.php

```diff
@@ -33,20 +31,10 @@ public function __construct(Config $config, Client $client = null)
         $this->exchange = new Exchange($this->config, $this->client);
     }
 
-    public function create(Payment $payment)
-    {
-        $body = $this->client->payment($this->getPaymentData($payment));
-
-        return $body;
-    }
-
-    public function request(Payment $payment)
-    {
-        $body = $this->client->request($this->getPaymentData($payment));
-
-        return $body;
-    }
-
+    /**
+     * @param  $country string Country
```

Comment:
Wrong order. Remember C:
`* @param string $country`
Also, as the description is the same as the variable name we can omit it.